Gemfile

```diff
@@ -17,4 +17,6 @@ use_logstash_source = ENV["LOGSTASH_SOURCE"] && ENV["LOGSTASH_SOURCE"].to_s == "
 if Dir.exist?(logstash_path) && use_logstash_source
   gem 'logstash-core', :path => "#{logstash_path}/logstash-core"
   gem 'logstash-core-plugin-api', :path => "#{logstash_path}/logstash-core-plugin-api"
-end
\ No newline at end of file
+end
+
+gem 'json', '>=2.3.0'
```

Comment:
I think as we are publishing this as a ruby gem, we should add these gem dependencies in gemspec file. But, I'm not sure though.

Reply:
I don't see it currently in https://github.com/opensearch-project/logstash-output-opensearch/blob/main/logstash-output-opensearch.gemspec. Should this be added explicitly with the version?

Reply:
yes, I don't see any gem dependency for json in gemspec file. Also, I found this:
`JSON is directly supported in Ruby, and has been since at least Ruby v1. 9.3, so there is no need to install a gem unless you're using something older. Simply use require 'json' in your code`

So, may be we don't need to add this gem dependency.

Reply:
It is safe to upgrade the json if old jruby is used to build this plugin. Like @naveentatikonda mentioned, lets add it as runtime dependency at gemspec file.

Reply:
@VijayanB @naveentatikonda I updated the PR b adding the json to the gemspec file.

Reply:
You can remove this in Gemfile.